src/main/resources/application.yml

```
@@ -27,4 +27,8 @@ spring:
     Properties:
       hibernate:
         format_sql: true # SQL 쿼리를 보기 좋게 포맷팅
-    database-platform: org.hibernate.dialect.PostgreSQLDialect
\ No newline at end of file
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+
+jwt:
+  secret: testKey
+  expiration: 86400000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)